models/api.models.js

```diff
@@ -0,0 +1,6 @@
+const db = require("../db/connection")
+
+exports.returnEndpoints = () => {
+    return db.query("SELECT table_name FROM INFORMATION_SCHEMA.TABLES WHERE table_type = 'BASE TABLE'").then((data) => {
```

Comment:
You dont really need this - Im not sure where it actually gets used as your just require in the json file in the controller.